hat/assets/js/apps/Iaso/domains/plannings/components/LinkToPlanning.tsx

```diff
@@ -0,0 +1,26 @@
+import React, { FunctionComponent } from 'react';
+
+import { Link } from 'react-router';
+import { userHasPermission } from '../../users/utils';
+import { baseUrls } from '../../../constants/urls';
+import { useCurrentUser } from '../../../utils/usersUtils';
+import { useGetPlanning } from '../../assignments/hooks/requests/useGetPlanning';
+
+type Props = {
+    planningId: string | undefined;
+    planningName: string;
```

Comment:
See above, it will ben a Planning type
anyway:
- `planningId: string | undefined; => planningId?: string;`
- Planning is required